### PR TITLE
Update Readme Credentials Section

### PR DESCRIPTION
**Pull request by nitro-neal**
update cred check marks

### CODE REVIEW

**Review by finn-tbd — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -315,9 +315,9 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 ## Presentation Definition V2
 | Feature               | Typescript | Kotlin | Rust | Swift |
 | --------------------- | ---------- | ------ | ---- | ----- |
-| Concrete Type         | ❌          | ❌      | ❌    | ❌     |
-| Validation            | ❌          | ❌      | ❌    | ❌     |
-| Credential Evaluation | ❌          | ❌      | ❌    | ❌     |
+| Concrete Type         | ✅          | ✅      | ❌    | ❌     |
+| Validation            | ✅          | ⚠️      | ❌    | ❌     |
```
Comment (decentralgabe) [inline README.md:319]: what does ⚠️ mean?

File: README.md [hunk 2]
```diff
@@ -291,10 +291,10 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 ## W3C Verifiable Credential Data Model 1.1
 | Feature             | Typescript | Kotlin | Rust | Swift |
 | ------------------- | ---------- | ------ | ---- | ----- |
-| Creation            | ❌          | ❌      | ❌    | ❌     |
-| Signing as `vc-jwt` | ❌          | ❌      | ❌    | ❌     |
-| Verification        | ❌          | ❌      | ❌    | ❌     |
-| Validation          | ❌          | ❌      | ❌    | ❌     |
+| Creation            | ✅          | ✅      | ❌    | ❌     |
+| Signing as `vc-jwt` | ✅          | ✅      | ❌    | ❌     |
+| Verification        | ✅          | ✅      | ❌    | ❌     |
```
Comment (decentralgabe) [inline README.md:296]: is verification also as vc-jwt? wondering if we can change the header to be VC-JWT

**Review by decentralgabe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mistermoe) [reply README.md:319]: image

File: README.md (hunk 2 above)
Comment (decentralgabe) [reply README.md:296]: yeah think it should all just be vc-jwt

**Pull request merged**